servers/rendering/renderer_rd/effects/fsr2.cpp

```diff
@@ -804,7 +804,9 @@ FSR2Effect::~FSR2Effect() {
 	RD::get_singleton()->free(device.linear_clamp_sampler);
 
 	for (uint32_t i = 0; i < FFX_FSR2_PASS_COUNT; i++) {
-		RD::get_singleton()->free(device.passes[i].pipeline.pipeline_rid);
+		if (device.passes[i].pipeline.pipeline_rid.is_valid()) {
+			RD::get_singleton()->free(device.passes[i].pipeline.pipeline_rid);
+		}
```

Comment:
You still have unrelated code, and another author not part of this, you will need to remove this code and reset the author of your commit with `git commit --reset-author`